.github/workflows/release.yml

```diff
@@ -0,0 +1,90 @@
+name: Create release
```

Comment:
@tjololo or anyone else who knows workflows better than me, is there any easy way to restrict this to only be able to run this from main branch?

Reply:
I dont't think you can restrict it from running. The workflow as it is will skip the checkout cdn and publish to cdn steps as far as I can see.

Reply:
Made some changes to this so none of the steps will run unless its on master. That should be good enough I think